From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MCOS <support@muthur-command.com>
Subject: [PATCH] neardi/lb200: run rk3576 arch_cpu_init in U-Boot proper

Vendor SPL configures SGRF for eMMC DMA, but mainline skipped the same
setup in U-Boot proper due to CONFIG_SPL_BUILD guard (use XPL_BUILD).
---
--- a/arch/arm/mach-rockchip/rk3576/rk3576.c
+++ b/arch/arm/mach-rockchip/rk3576/rk3576.c
@@ -151,7 +151,7 @@ int arch_cpu_init(void)
 {
 	u32 val;
 
-	if (!IS_ENABLED(CONFIG_SPL_BUILD))
+	if (IS_ENABLED(CONFIG_XPL_BUILD))
 		return 0;
 
 	/* Set the emmc to access ddr memory */
